scripts/make/build.mk

```diff
@@ -52,4 +52,16 @@ $(OUT_DIR):
 $(OUT_BIN): _cargo_build $(OUT_ELF)
 	$(call run_cmd,$(OBJCOPY),$(OUT_ELF) --strip-all -O binary $@)
 
+ifeq ($(ARCH), aarch64)
+  uimg_arch := arm64
+else ifeq ($(ARCH), riscv64)
+  uimg_arch := riscv
+else
+  uimg_arch := $(ARCH)
+endif
+uimg_kernel_base := $(subst _,,$(shell axconfig-gen "configs/platforms/$(PLATFORM).toml" -r plat.kernel-base-paddr))
```

Comment:
`axconfig-gen "$(OUT_CONFIG)"`

Reply:
There is not the $(OUT_CONFIG) .yml file when building first.